src/libfetchers/git-utils.cc

```diff
@@ -205,6 +205,24 @@ static git_packbuilder_progress PACKBUILDER_PROGRESS_CHECK_INTERRUPT = &packBuil
 
 } // extern "C"
 
+static void initRepoAtomically(std::filesystem::path &path, bool bare) {
+    Path tmpDir = createTempDir(std::filesystem::path(path).parent_path());
+    AutoDelete delTmpDir(tmpDir, true);
+    Repository tmpRepo;
+    if (git_repository_init(Setter(tmpRepo), tmpDir.c_str(), bare))
+        throw Error("creating Git repository %s: %s", path, git_error_last()->message);
```

Comment:
```suggestion
        throw Error("creating Git repository '%s': %s", path, git_error_last()->message);
```

Reply:
I believe this would get quoted twice, see my error message above.